pkg/users/users.js

```diff
@@ -153,15 +161,33 @@ function AccountsPage() {
     } else if (path.length === 1) {
         return (
             <AccountDetails accounts={accountsInfo} groups={groupsExtraInfo}
-                            current_user={current_user_info?.name} user={path[0]} shells={shells} />
+                current_user={current_user_info?.name} user={path[0]} shells={shells} />
         );
     } else return null;
 }
 
-async function getLogins() {
+async function getLastlogPath() {
+    try {
+        await cockpit.spawn(["test", "-e", "/var/run/utmp"], { err: "ignore" });
+        return "/var/run/utmp";
+    } catch (err1) {
+        try {
+            await cockpit.spawn(["test", "-e", "/var/lib/lastlog/lastlog2.db"], { err: "ignore" });
+            return "/var/lib/lastlog/lastlog2.db";
+        } catch (err2) {
+            return null;
+        }
+    }
+}
+
+async function getLogins(LastLogVersion) {
     let lastlog = "";
     try {
-        lastlog = await cockpit.spawn(["lastlog"], { environ: ["LC_ALL=C"] });
+        if (LastLogVersion === 1) {
+            lastlog = await cockpit.spawn(["lastlog"], { environ: ["LC_ALL=C"] });
+        } else {
+            lastlog = await cockpit.spawn(["lastlog2"], { environ: ["LC_ALL=C"] });
```

Comment:
`lastlog2` is a drop-in replacement for `lastlog` so you can always just call `lastlog`. There's no need to differentiate between the two.

Reply:
On my Tumbleweed machine I have lastlog2 but not lastlog

Reply:
My bad. Turns out that the symlink was in fact removed from tumbleweed and it's unlikely to come back.
(for context: https://lists.opensuse.org/archives/list/factory@lists.opensuse.org/thread/NUT6SYUETXQ4PW2L26SEIGO3V7U4F3ZX/)